hal/audio_extn/Android.mk

```diff
@@ -381,6 +381,10 @@ LOCAL_SRC_FILES:= \
         cirrus_playback.c
 endif
 
+ifeq ($(strip $(AUDIO_FEATURE_SONY_CIRRUS_CALIB_TA)),true)
+LOCAL_CFLAGS += -DGET_SPEAKER_CALIBRATIONS_FROM_TA
```

Comment:
Not sure why we need this though, because we do the calibration ourselves and read this back from the mixer.

Reply:
The `cirrus_sony` module would first load TA calibration data then save the calibration to "/data/vendor/audio/cirrus_sony.cal" for future use. So I think the calibration data should first be loaded from TA?

This also matches the behavior of stock, as found in `audio.primary.lahaina.so` in stock version 61.2.A.0.388 (Decompiled using Ghidra), it will load calibration data from TA unit 4700 - 4714. So I would assume that the calibration data is already here when the device is shipped from factory.

Reply:
But we are not stock are we?

> Decompiled using Ghidra

Don't.

Reply:
What I want to argue is that if sony already saves the calibration in TA partition, why don't we use it?